### PR TITLE
Seed/abd create prompts

### LINKED ISSUE #15: Add seeds
- [x] authors
- [x] stories
- [ ] prompts

### PR DESCRIPTION
**Pull request by Abdeboskey**
#### What's this PR do?
* Create the initial seed file for the writing prompts
#### Where should the reviewer start?
* Start at prompts-seed.js
#### How should this be manually tested?
* Pull this branch, run `knex migrate:latest` and then run `knex seed:run`
* Open Postico, and examine the database. 
#### Any background context you want to provide?
* Began making commits using a ticket number from an issue on the front end (#31), Twas silly, and I course-corrected to tag issue #15 
#### What are the relevant tickets?
closes #15 
#### Screenshots (if appropriate)
Screen Shot 2020-09-13 at 5 33 32 PM

#### Questions:
* I may need help getting the correct genres for all of these, I think they mostly make sense but please let me know if there are any genres you would adjust. 
* If there are any prompts you don't like, please let me know and we can remove/adjust them.

### CODE REVIEW

**Review by nickhartdev — APPROVED** [review]
Only read through the first half of prompts, and I'm a fan. Thanks for formatting these!

**Review by carlymclift — APPROVED** [review]
Love these prompts! this is going to be a lot of fun!

File: db/seeds/dev/prompts_seed.js [hunk 1]
```diff
@@ -4,10 +4,430 @@ exports.seed = function(knex) {
...
+        {
+          id: 2,
+          prompt: "Your mom was an alien the whole time",
+          genre: "science fiction",
+        },
+        {
+          id: 3,
+          prompt: "The president is a really nice guy",
+          genre: "satire",
+        },
```
[the remark this answers is not in the document]
Comment (GreysonElkins) [reply db/seeds/dev/prompts_seed.js:22]: Maybe number 2, and number 1 is in 2nd place, number 3 is dumb. Don't care if they live-on though!

File: db/seeds/dev/prompts_seed.js [hunk 2]
```diff
@@ -4,10 +4,430 @@ exports.seed = function(knex) {
...
+        {
+          id: 14,
+          prompt:
+            'Wikipedia is shut down and all copies deleted for lack of funds and loss of net neutrality. This is the founder\'s "I warned you, jerks" notification.',
+          genre: "dystopian",
+        },
+        {
+          id: 15,
+          prompt:
+            "In a world where puns are illegal, one man rises up in opposition.",
```
[the remark this answers is not in the document]
Comment (GreysonElkins) [reply db/seeds/dev/prompts_seed.js:92]: Second that

**Review by GreysonElkins — APPROVED** [review]
This looks great! Approved for moving forward - happy to take a closer look at genres another time if it's felt to be needed!

Thanks for doing this @Abdeboskey

**Pull request merged**